modules/README.md

```diff
@@ -0,0 +1,29 @@
+# Modules
+
+## Overview
+
+Modules are the next generation of components in Kyma that are available for local and cluster installation.
+
+Modules are no longer represented by a single helm-chart, but instead are bundled and released within channels through a `ModuleTemplate`, a unique link of a module, and its desired state of charts and configuration, and a channel.
+
+This directory includes public `ModuleTemplates` for all channels available for consumption within Kyma, wherein each channel is represented by a dedicated kustomization.
+
+## Development
+
+To apply all `ModuleTemplates` for your cluster, use the kustomization available in this folder:
+
+```
+kubectl apply -k modules
+```
+
+Then you can enable available Modules either with support of Busola through the `Kyma` Custom Resource, or you can manually edit the `Kyma CR` for the same effect.
+
+## Adding a new Module
+
+The only type of Module currently accepted is `ModuleTemplate` in version `v1alpha1`.
```

Comment:
Wouldn't it be a good idea to provide links to some `ModuleTemplate` and `v1alpha1` documentation?

Reply:
ModuleTemplates are autogenerated in CLI with `kyma alpha module create` but since this command is still alpha, I didn't want to link it here. WDYT?

Reply:
> ModuleTemplates are autogenerated in CLI with `kyma alpha module create` but since this command is still alpha, I didn't want to link it here. WDYT?
Maybe you could add a NOTE explaining that for the time being `kyma alpha module create` is the only available option. It could be removed/ replaced later. Otherwise, the instructions seem to be somewhat incomplete, don't you think?